Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/E2ETests/E2EAnalysisTests.cs

```diff
@@ -826,7 +826,7 @@ public void E2E_TestProjects_ProtobufsUpdated()
         // Assert
         result.AssertTargetExecuted("FixUpTestProjectOutputs");
 
-        var protobufDir = Path.Combine(result.GetPropertyValue("ProjectSpecificOutDir"), "subdir1");
```

Comment:
MsBuild structured logger has an issue when reading the bin files generated by .NET 9 and the property names got changed. I extracted it to a variable to ease the update when the problem will be addressed.